searchPage/search.css

```diff
@@ -1,27 +1,152 @@
-@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,900;1,700&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,900;1,700&display=swap");
 
 * {
-    padding: 0;
-    margin: 0;
-  }
-  header{
-    display: flex;
-    justify-content: center;
-   }
-   .header{
-     width: 90%;
-     display: flex;
-     justify-content: space-between;
- margin-top: 3.7vh;
- 
- }
- #logo{
-   font-family: 'Roboto', sans-serif;
-   color: #4096df;
-   
-   font-size: 1.5rem;
- }
- 
- .menuIcon{
-   font-size: 1.5rem;
- }
\ No newline at end of file
+  padding: 0;
+  margin: 0;
+  font-family: "Montserrat", sans-serif;
+  box-sizing: border-box;
+}
+header {
+  display: flex;
+  justify-content: center;
+}
+.header {
+  width: 90%;
+  display: flex;
+  justify-content: space-between;
+  margin-top: 3.7vh;
+}
+#logo {
+  font-family: "Roboto", sans-serif;
+  color: #4096df;
+  font-size: 1.5rem;
+}
+
+#menuIcon {
+  font-size: 1.5rem;
+}
+
+#image {
+  margin-bottom: 0%;
+  margin-top: 1.5rem;
+}
+
+#ima {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 0px;
+}
+
+#most {
```

Comment:
the size of the font is big, and you need to decrease left margin and increase the top one